BattleNetwork/netplay/bnPacketShipper.cpp

```diff
@@ -81,37 +82,26 @@ std::pair<Reliability, uint64_t> PacketShipper::Send(
     break;
   // (Specialized Reliability::Reliable) handles chunking big packets
   case Reliability::BigData:
-    size_t max_buffer_size = (socket.getSendBufferSize() / sizeof(char)) / 2; // bytes len -> list len
-    size_t body_size = body.size();
-    size_t header_size = sizeof(nextReliable) + sizeof(size_t)*2 + sizeof(int); // headers 1-4 below
+    size_t bodySize = body.size();
+    size_t headerSize = 1 + sizeof(nextReliable) + sizeof(size_t) * 2; // headers 1-4 below
+    size_t maxChunkSize = maxPayloadSize - headerSize;
 
-    size_t expected_chunks = body_size / max_buffer_size;
-    size_t remainder = body_size % max_buffer_size;
+    size_t expectedChunks = bodySize / maxChunkSize;
+    size_t remainder = bodySize % maxChunkSize;
```

Comment:
note to self: check if recalculation of packets to ship is needed after accounting for the required headers

Reply:
No, by removing the header cost from the max chunk size, we shrink the available outgoing space and calculate the number of outgoing packets effectively skipping the need to recalculate an inflated outgoing sum.